### ISSUE TITLE
[JENKINS-12474] Subversion rejected Basic challenge

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/subversion-plugin#898]
<p>After setting up a fresh jenkins project for subversion I got the following error messages on the console:</p>

<p><span class="error">&#91;INFO&#93;</span> Executing: cmd.exe /X /C "svn --non-interactive update c:\jenkins\jobs\XmlTransformer_trunk\workspace"<br/>
<span class="error">&#91;INFO&#93;</span> Working directory: c:\jenkins\jobs\XmlTransformer_trunk\workspace<br/>
Provider message:<br/>
The svn command failed.<br/>
Command output:<br/>
svn: OPTIONS of 'http://developbender/svn/dev/Sonic/bendit/Libraries/XmlTransformer/trunk': authorization failed: Could not authenticate to server: rejected Basic challenge (<a href="http://developbender" class="external-link" target="_blank" rel="nofollow noopener">http://developbender</a>)</p>

<p>I know that there are thousands of such reports, but I have tried everything to resolve this one without luck. Please note that I have installed hudson in parallel in the same Tomcat, which works perfectly without having this problem!<br/>
I could only assume that this might have to do with the earlier reported bug: <a href="https://issue-redirect.jenkins.io/issue/12473" title="Subversion option vanishes under Source-Code-Management in project configuration" class="issue-link" data-issue-key="<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/12473">JENKINS-12473</a>"><del>JENKINS-12473</del></a> so that Jenkins does not give the credentials to subversion here.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=powerstat">powerstat</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-12474" target="_blank">Subversion rejected Basic challenge</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=recena">recena</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Blocker
<li><b>component(s)</b>: subversion-plugin
<li><b>label(s)</b>: credentials
<li><b>resolution</b>: Won't Do
<li><b>resolved</b>: 2015-11-09T10:16:24+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 20251212-090250
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>After setting up a fresh jenkins project for subversion I got the following error messages on the console:</p>

<p><span class="error">&#91;INFO&#93;</span> Executing: cmd.exe /X /C "svn --non-interactive update c:\jenkins\jobs\XmlTransformer_trunk\workspace"
<span class="error">&#91;INFO&#93;</span> Working directory: c:\jenkins\jobs\XmlTransformer_trunk\workspace
Provider message:
The svn command failed.
Command output:
svn: OPTIONS of 'http://developbender/svn/dev/Sonic/bendit/Libraries/XmlTransformer/trunk': authorization failed: Could not authenticate to server: rejected Basic challenge (<a href="http://developbender" class="external-link" target="_blank" rel="nofollow noopener">http://developbender</a>)</p>

<p>I know that there are thousands of such reports, but I have tried everything to resolve this one without luck. Please note that I have installed hudson in parallel in the same Tomcat, which works perfectly without having this problem!
I could only assume that this might have to do with the earlier reported bug: <a href="https://issues.jenkins.io/browse/JENKINS-12473" title="Subversion option vanishes under Source-Code-Management in project configuration" class="issue-link" data-issue-key="JENKINS-12473"><del>JENKINS-12473</del></a> so that Jenkins does not give the credentials to subversion here.</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>Windows 2003 Server, Java 1.6.0_29, Tomcat 7, Fresh installed Jenkins</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> recena:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-12474?focusedId=240335&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-240335">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=powerstat" class="user-hover" rel="powerstat">powerstat</a>, Which version of Jenkins and Subversion Plugin are you using?</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=powerstat" class="user-hover" rel="powerstat">powerstat</a>, Which version of Jenkins and Subversion Plugin are you using?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> recena:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-12474?focusedId=241184&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-241184">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Subversion Plugin does not use with Subversion Client in CLI. Therefore, this issue is not related to Subversion Plugin.</p>


<div class="preformatted panel" style="border-width: 1px;"><div class="preformattedContent panelContent">
<pre>[INFO] Executing: cmd.exe /X /C "svn --non-interactive update c:\jenkins\jobs\XmlTransformer_trunk\workspace"
[INFO] Working directory: c:\jenkins\jobs\XmlTransformer_trunk\workspace
Provider message:
The svn command failed.
Command output:
svn: OPTIONS of 'http://developbender/svn/dev/Sonic/bendit/Libraries/XmlTransformer/trunk': authorization failed: Could not authenticate to server: rejected Basic challenge (http://developbender)
</pre>
</div></div>
</pre>
</details>

</ul>
</details>

<p>Subversion Plugin does not use with Subversion Client in CLI. Therefore, this issue is not related to Subversion Plugin.</p>




```
[INFO] Executing: cmd.exe /X /C "svn --non-interactive update c:\jenkins\jobs\XmlTransformer_trunk\workspace"
[INFO] Working directory: c:\jenkins\jobs\XmlTransformer_trunk\workspace
Provider message:
The svn command failed.
Command output:
svn: OPTIONS of 'http://developbender/svn/dev/Sonic/bendit/Libraries/XmlTransformer/trunk': authorization failed: Could not authenticate to server: rejected Basic challenge (http://developbender)

```

**Comment by jenkins-infra-bot** [comment]
<i>[Original `relates_to` from Jira: <a class="jira-relationship" href="https://issue-redirect.jenkins.io/issue/4919">JENKINS-4919</a>]</i>
